src/System.Runtime.Extensions/src/ApiCompatBaseline.net462.txt

```diff
@@ -0,0 +1 @@
+MembersMustExist : Member 'System.IO.Path.CreateRelativePath(System.String, System.String)' does not exist in the implementation but it does exist in the contract.
```

Comment:
We can't have exceptions for net462, that's already shipped.  I suspect that the net462 targetgroup is resolving the wrong contract assembly.